sormas-backend/src/test/java/de/symeda/sormas/backend/caze/CaseFacadeEjbTest.java

```diff
@@ -1844,4 +1847,52 @@ public void testCaseCriteriaLastShareWithReportingToolBetweenDates() {
 		MatcherAssert.assertThat(indexList, hasSize(0));
 
 	}
+
+	@Test
+	public void testCaseCompletenessWhenCaseFound() {
+		RDCF rdcf = creator.createRDCF();
+		UserDto user = creator.createUser(rdcf, UserRole.NATIONAL_USER);
+
```

Comment:
Run the method before a case is present to see if it finds zero cases to update.

Reply:
updated